packages/rpc-provider/src/ws/Provider.ts

```diff
@@ -69,7 +69,7 @@ export default class WsProvider implements WSProviderInterface {
   private autoConnect: boolean;
   private coder: Coder;
   private endpoint: string;
-  private handlers: {
+  public handlers: {
```

Comment:
I still think these should be private, end-users touch WsProvider, and they shouldn't care what handlers are.

To silent the TS accessing private methods, just put @ts-ignore and a small comment

Reply:
Ok, I did that for all the tests the need to access the `handlers`, `websocket` properties and the `onSocketMessage` method